com.toxicstoxm.ledsuite.yml

```diff
@@ -0,0 +1,63 @@
+id: com.toxicstoxm.ledsuite
+branch: '0.1.3'
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk
+finish-args:
+  - --share=network
+  - --socket=wayland
+  - --share=ipc
+  - --socket=fallback-x11
+  - --device=dri
+#build-options:
+#  build-args:
+#    - --share=network
+command: LEDSuite
+
+modules:
+
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk/install.sh
+      - mkdir /app/bin
+      - ln -s /app/jre/bin/java /app/bin/java
+
+  - name: ledsuite
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk/gradle/bin/gradle build
+      - install -Dm644 build/libs/LEDSuite-*-final.jar /app/LEDSuite.jar
+      - install -Dm755 LEDSuite.sh /app/bin/LEDSuite
+
+    sources:
+      - type: archive
+        url: https://github.com/ToxicStoxm/LEDSuite/archive/refs/tags/v0.1.3.tar.gz
+        sha256: 680911586cf67e9f5d654f9b874899809bf06298416defd4f4843d0bdaee25b7
+
+      - type: script
+        dest-filename: LEDSuite.sh
+        commands:
+          - java --enable-native-access=ALL-UNNAMED -jar /app/LEDSuite.jar $@
+
+      - flatpak-sources.json # All the dependencies for the app (generated using "gradle flatpakGradleGenerator")
+
+  - name: appdata
```

Comment:
this should be part of the main module.

Reply:
What should be part of the main module? The flatpak-sources or appdata?

Reply:
![image](https://github.com/user-attachments/assets/b3c9561a-96a0-47c3-b897-1629c9b5d493)

I don't see what's ambiguous?

Reply:
Moved appdata module contents into main module.